tests/cluster_tests/run_mpi_test.sh

```diff
@@ -2,6 +2,8 @@
 
 cd $PBS_O_WORKDIR/
 
-module load raven-devel-gcc
+module load raven-devel
+module load MVAPICH2/2.0.1-GCC-4.9.2
+source activate raven_libraries
```

Comment:
Is conda activate available?

Reply:
nope.

Reply:
Defining the "conda" functions within a script is challenging (see establish_conda_env.sh) and often isn't worth it.  For now, using "source activate" is more efficient; hopefully by Conda 4.6 (or later) the conda devs figure out how they want us to do things; right now both are acceptable.